nvr/nvr.py

```diff
@@ -330,20 +330,33 @@ def main(argv=sys.argv, env=os.environ):
         print_sockaddrs()
         return
 
-    address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or '/tmp/nvimsocket'
+    if sys.platform == 'win32':
+        address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or None
+        if address == None:
+            print("Windows workaround: could not find an address! Set environment variable 'NVIM_LISTEN_ADDRESS', or pass a server address when calling nvr: `nvr --servername \\some\\server\\address`.")
+            # possibility: what could be a good temp file location on windows? Once we decide on this, we could use the subprocess possibility described below to start an nvim process on windows using this temp server. 
+            sys.exit(1)
+    else:
+        address = options.servername or env.get('NVIM_LISTEN_ADDRESS') or '/tmp/nvimsocket'
 
     nvr = Nvr(address, options.s)
     nvr.attach()
 
     if not nvr.server:
-        nvr.address = sanitize_address(address)
-        silent = options.remote_silent or options.remote_wait_silent or options.remote_tab_silent or options.remote_tab_wait_silent or options.s
-        if not silent:
-            show_message(address, nvr.address)
-        if options.nostart:
+        if sys.platform == 'win32':
+            print("Windows workaround: nvr on Windows can only attach to existing neovim processes! Could not attach to supplied server address: {}.".format(address))
             sys.exit(1)
+            # possibility: use user supplied address to start an nvim process as subprocess (see https://docs.python.org/3.6/library/subprocess.html)
+            # but I don't know how to start nvim on windows with such an address: https://vi.stackexchange.com/questions/14883/how-do-i-set-nvim-listen-address-on-startup-using-windows
```

Comment:
`nvr.start_new_process()` sets the value in the environment, already.  There's no need to try doing it as part of the command.

Reply:
So, have something like:

    if sys.platform == 'win32':
            os.environ['NVIM_LISTEN_ADDRESS'] = self.address
            try:
                args = os.environ.get('NVR_CMD')
                args = args.split(' ') if args else ['nvim']
                os.dup2(sys.stdout.fileno(), sys.stdin.fileno())
                os.execvpe(args[0], args, os.environ)
            except FileNotFoundError:
                print("[!] Can't start new nvim process: '{}' is not in $PATH.".format(args[0]))
                sys.exit(1)

Reply:
For me though, `os.environ.get('NVR_CMD')` is `None`? (Windows 10)